sky/backends/cloud_vm_ray_backend.py

```diff
@@ -723,6 +723,16 @@ def _update_blocklist_on_gcp_error(
                 logger.info('Skipping all regions due to disk size issue.')
                 self._blocked_resources.add(
                     launchable_resources.copy(region=None, zone=None))
+            elif ('Policy update access denied.' in httperror_str[0] or
+                  'IAM_PERMISSION_DENIED' in httperror_str[0]):
+                logger.info('Skipping all regions due to service account not '
+                            'having the required permissions and the user '
+                            'account does not have enough permission to '
+                            'update it. Please contact your administrator and '
+                            'check out: https://skypilot.readthedocs.io/en/latest/cloud-setup/cloud-permissions.html#gcp')  # pylint: disable=line-too-long
```

Comment:
Can we print the detailed underlying error too?

Reply:
Good point! Fixed. Thanks!